rsmtool/notebooks/header.ipynb

```diff
@@ -61,10 +61,8 @@
+    "javascript_path = os.environ.get(\"JAVASCRIPT_PATH\")\n",
```

Comment:
Perhaps it would be best to keep all os.environ.get commands together? This way if somebody wants to set them manually, they are all in the same place?